src/cmd/package.go

```diff
@@ -156,33 +156,37 @@ func cachePathClean(cachePath string) bool {
 }
 
 func init() {
+	initViper()
+
 	rootCmd.AddCommand(packageCmd)
 	packageCmd.AddCommand(packageCreateCmd)
 	packageCmd.AddCommand(packageDeployCmd)
 	packageCmd.AddCommand(packageInspectCmd)
 	packageCmd.AddCommand(packageRemoveCmd)
 	packageCmd.AddCommand(packageListCmd)
 
+	v.SetDefault("package.create.zarf_cache", config.ZarfDefaultImageCachePath)
+
+	//  Always require confirm flag (no viper)
 	packageCreateCmd.Flags().BoolVar(&config.CommonOptions.Confirm, "confirm", false, "Confirm package creation without prompting")
-	packageCreateCmd.Flags().StringVar(&config.CommonOptions.TempDirectory, "tmpdir", "", "Specify the temporary directory to use for intermediate files")
-	packageCreateCmd.Flags().StringToStringVar(&config.CommonOptions.SetVariables, "set", map[string]string{}, "Specify package variables to set on the command line (KEY=value)")
-	packageCreateCmd.Flags().StringVar(&zarfImageCache, "zarf-cache", config.ZarfDefaultImageCachePath, "Specify the location of the Zarf image cache")
-	packageCreateCmd.Flags().StringVarP(&config.CreateOptions.OutputDirectory, "output-directory", "o", "", "Specify the output directory for the created Zarf package")
-	packageCreateCmd.Flags().BoolVar(&config.CreateOptions.SkipSBOM, "skip-sbom", false, "Skip generating SBOM for this package")
-	packageCreateCmd.Flags().BoolVar(&config.CreateOptions.Insecure, "insecure", false, "Allow insecure registry connections when pulling OCI images")
+	packageCreateCmd.Flags().StringToStringVar(&config.CommonOptions.SetVariables, "set", v.GetStringMapString("package.create.set"), "Specify package variables to set on the command line (KEY=value)")
+	packageCreateCmd.Flags().StringVar(&zarfImageCache, "zarf-cache", v.GetString("package.create.zarf_cache"), "Specify the location of the Zarf image cache")
+	packageCreateCmd.Flags().StringVarP(&config.CreateOptions.OutputDirectory, "output-directory", "o", v.GetString("package.create.output_directory"), "Specify the output directory for the created Zarf package")
+	packageCreateCmd.Flags().BoolVar(&config.CreateOptions.SkipSBOM, "skip-sbom", v.GetBool("package.create.skip_sbom"), "Skip generating SBOM for this package")
+	packageCreateCmd.Flags().BoolVar(&config.CreateOptions.Insecure, "insecure", v.GetBool("package.create.insecure"), "Allow insecure registry connections when pulling OCI images")
 
+	//  Always require confirm flag (no viper)
 	packageDeployCmd.Flags().BoolVar(&config.CommonOptions.Confirm, "confirm", false, "Confirm package deployment without prompting")
-	packageDeployCmd.Flags().StringVar(&config.CommonOptions.TempDirectory, "tmpdir", "", "Specify the temporary directory to use for intermediate files")
 	packageDeployCmd.Flags().StringToStringVar(&config.CommonOptions.SetVariables, "set", map[string]string{}, "Specify deployment variables to set on the command line (KEY=value)")
```

Comment:
Was this overlooked for deploy? I see the package.create.set, but no package.deploy.set.

Reply:
fixed thanks!